### PR TITLE
Issue website add lighthouse check

### LINKED ISSUE #740: add lighthouse check for websites
## Is your feature request related to a problem? Please describe

We currently get surprised when our metrics get worse.

## Describe the solution you'd like

use https://www.npmjs.com/package/lighthouse to compare between the live site and the current site in the runner and fail if any of top level points is worse than the live score.

### PR DESCRIPTION
**Pull request by owais34**
# The Pull Request is ready

- [x] fixes #740 
- [x] all actions are passing
- [x] only fixes a single issue

## Overview

Added script to include lighthouse check , comparing high level points for current website and live website.

## Documentation-Website

- [ ] mobile view is usable
- [ ] desktop view is usable
- [ ] no a-tags are used directly (NavLink, MailLink, ExternalLink instead)
- [ ] all new texts are added to the translation files (at least the english one)
- [ ] tests have been added (if required)
- [ ] shared code has been extracted in a different file

### CODE REVIEW

File: documentation-website/tools/lighthouse-report.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      // eslint-disable-next-line max-len
+      `${ metricField.name } score reduced to ${ devMetricScore } from ${ prodMetricScore }`,
+    );
+    // eslint-disable-next-line no-magic-numbers
+    process.exit(1,);
+  }
+}
+
+// eslint-disable-next-line no-magic-numbers
+process.exit(0,);
```
[the remark this answers is not in the document]
Comment (Idrinth) [reply documentation-website/tools/lighthouse-report.js:64]: we should have a constants file with them already, if not please create one!

**Review by Idrinth — CHANGES_REQUESTED** [review]

**Comment by owais34** [comment]
@Idrinth  , got it I'll resolve above issues 👍

**Comment by owais34** [comment]
@Idrinth  , there seems to be some issue with the server run scripts.
Ive tried : 
1. npm start
2. npm serve
The server is hosted , but upon visiting the url a blank page is displayed. The dist/index.html page doesn't have any script tags.
Only npm run dev , seems to host the website locally.

**Comment by Idrinth** [comment]
> @Idrinth , there seems to be some issue with the server run scripts. Ive tried :
> 
>     1. npm start
> 
>     2. npm serve
> 
> 
> The server is hosted , but upon visiting the url a blank page is displayed. The dist/index.html page doesn't have any script tags. Only npm run dev , seems to host the website locally.

On linux(the actions run on linux) that works :( odd, will try to figure out what goes wrong there

**Review by Idrinth — APPROVED** [review]

**Comment by Idrinth** [comment]
thanks a lot @owais34 :)

**Comment by owais34** [comment]
@Idrinth , you're welcome sir ;).

**Pull request merged**